orchestrators/kubernetes.go

```diff
@@ -297,3 +300,12 @@ func (o *KubernetesOrchestrator) getConfig() (config *rest.Config, err error) {
 	}
 	return
 }
+
+func createWorkerName() string {
+	var letter = []rune("abcdefghijklmnopqrstuvwxyz0123456789")
+	b := make([]rune, 20)
+	for i := range b {
+		b[i] = letter[rand.Intn(len(letter))]
+	}
+	return "conplicity-" + string(b)
+}
```

Comment:
That doesn't guarantee unicity but it's probably good enough if we're lucky :wink:

Reply:
And what about that: cbedac7

Reply:
couldn't it take lot of time on big environments with many Pods to list ? just asking with knowledge (yet).

Reply:
Good question. It would be better to set an exception when creating the pod. If the creation fails because of the name, then try again with a new one.